Python/getargs.c

```diff
@@ -2831,6 +2831,24 @@ _PyArg_NoPositional(const char *funcname, PyObject *args)
     return 0;
 }
 
+int
+_PyArg_NoTupleKeywords(const char *funcname, PyObject *kwnames)
+{
+    if (kwnames == NULL) {
+        return 1;
+    }
+
+    assert(PyTuple_CheckExact(kwnames));
+
+    if (PyTuple_GET_SIZE(kwnames) == 0) {
+        return 1;
+    }
+
+    PyErr_Format(PyExc_TypeError, "%.200s() takes no keyword arguments",
```

Comment:
The %.200s is an old habbit from the 90s which is no longer needed. PyErr_Format() is able to allocate enough memory for longer function name.

```suggestion
    PyErr_Format(PyExc_TypeError, "%s() takes no keyword arguments",
```